tests/unit/account_info_test.py

```
@@ -65,6 +70,32 @@ def test_account_info_initialization(account_info):
     assert account_info.token_relationships == []
     assert account_info.account_memo == "Test account memo"
     assert account_info.owned_nfts == 5
+    assert account_info.max_automatic_token_associations == 10
+    assert account_info.staking_info is None
+
+def test_from_proto_with_staking_info():
+    """Test the from_proto method of the AccountInfo class with staking info"""
+    public_key = PrivateKey.generate_ed25519().public_key()
+    
+    staking_info={
```

Ruff (0.14.14)

[error] Local variable `staking_info` is assigned to but never used

Remove assignment to unused variable `staking_info`

(F841)